### PR TITLE
27 rootfs directory tree simplification

### PR DESCRIPTION
**Pull request by daniel-rossier**

### CODE REVIEW

File: target/rpi4_64_so3_standalone.its [hunk 1]
```diff
@@ -25,33 +25,33 @@
...
 			description = "SO3 OS kernel";
-			data = /incbin/("../so3/so3.bin");
+			data = /incbin/("../so3/so3/so3.bin");
 			type = "kernel";
 			arch = "arm64";
 			os = "linux";
 			compression = "none";
-			load = <0x41008000>;
-			entry = <0x41008000>;
+            load = <0x01080000>;
```
Comment (jmi-reds) [inline target/rpi4_64_so3_standalone.its:33]: alignement

Comment (daniel-rossier) [reply target/rpi4_64_so3_standalone.its:33]: Done

Comment (jmi-reds) [reply target/rpi4_64_so3_standalone.its:33]: I don't see the changes.

Comment (daniel-rossier) [reply target/rpi4_64_so3_standalone.its]: Thx and sorry, forgot to push :-)

**Review by jmi-reds — APPROVED** [review]

**Pull request merged**